### PR TITLE
Add .alfredworkflow file

### PR DESCRIPTION
**Pull request by Klaudioz**
I just added the file I commented you on Twitter.

### CODE REVIEW

**Comment by kevinjalbert** [comment]
@Klaudioz I'm sorry for taking so long to getting to this PR.

I do appreciate the PR, although at this point I think I'll have to come up with a better approach for creating releases with the `.alfredworkflow` file. It requires all the script files, otherwise, it doesn't work as expected (I checked yours and it was just the `info.plist` in there).

I had the workflow file in the releases for a while, but I took it away after a slight data leak was found out (https://github.com/kevinjalbert/alfred-notion/commit/bbb253015bd92124c0ccab9d8ce6180c2c15169d).

-----

### TL;DR

I really appreciate the PR.

I'm going to close this PR out and make an issue to actually come up with a better release process.
